src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/StandardResolver.cs

```diff
@@ -283,6 +283,7 @@ internal static class StandardResolverHelper
         public static readonly IFormatterResolver[] DefaultResolvers = new IFormatterResolver[]
         {
             BuiltinResolver.Instance, // Try Builtin
+            StringInterningResolver.Instance, // intern strings if enabled.
```

Comment:
Since BuiltinResolver includes a Formatter for strings, this line is meaningless.
(The unit tests used the Formatter directly, so the StandardResolver priority check was not included.)

Reply:
Fixed.